altair/vegalite/v5/api.py

```diff
@@ -3865,101 +3850,81 @@ def interactive(  # noqa: C901
             encodings.append("x")
         if bind_y:
             encodings.append("y")
-        interactive_chart = self.add_params(
+        chart: Chart = self.copy().add_params(
```

Comment:
Moved the `.copy()` to occur first, otherwise the params are added to the original chart as well